core/examples/luigi-sample-angular/src/luigi-config/extended/auth.js

```diff
@@ -20,6 +20,14 @@ class Auth {
     authorizeMethod: 'GET',
     oAuthData: {
       client_id: 'egDuozijY5SVr0NSIowUP1dT6RVqHnlp'
+    },
+    userInfoFn: () => {
```

Comment:
userInfoFn should give an input to the user. eg. the authorization data., how would you get the `Luigi User`?

Reply:
Ah, if using mock  .. it is only using the provider's `userInfo`, but ignoring a provided `userInfoFn`. 
Maybe we change the implementation to allow to override the default provider userInfo function.